corehq/apps/export/models/new.py

```diff
@@ -2772,6 +2769,15 @@ class EmailExportWhenDoneRequest(models.Model):
     user_id = models.CharField(max_length=255)
 
 
+class LedgerSectionEntry(models.Model):
```

Comment:
what do you think about bootstrapping this table with data from ES?

Reply:
Yeah. That's what I meant by the TODO in the description. implemnted here: https://github.com/dimagi/commcare-hq/pull/25304/commits/f9184cf162e4005d5ca3ff975bbac0250892b994